proposals/low-level-struct-improvements.md

```diff
@@ -309,6 +309,13 @@ The span safety rules will be written in terms of `scoped ref` and `ref`. For sp
 
 When discussing the *ref-safe-to-escape* of arguments that correspond to `in` parameters they will be generalized as `ref` arguments in the spec. In the case the argument is an lvalue then the *ref-safe-to-escape* is that of the lvalue, otherwise it is *current method*. Again `in` will only be called out here when it is important to the semantic of the current rule.
 
+<a name="return-only"></a>
+The design also requires that the introduction of a new escape scope: *return only*. This is similar to *calling method* in that it can be returned but it can **only** be returned through a `return` statement. 
+
+The details of *return only* is that it's a scope which is greater than *current method* but smaller than *calling method*. An expression provided to a `return` statement must be at least *return only*. As such most existing rules fall out. For example assignment into a `ref` parameter from an expression with a *safe-to-escape* of *return only* will fail because it's smaller than the `ref` parameters *safe-to-escape* which is *calling method*. The need for this new escape scope will be discussed [below](#rules-unscoped). 
```

Comment:
>parameters [](http://example.com/codeflow?start=393&length=10)

typo? missing apostrophy in "parameter's" in "it's smaller than the `ref` parameter's *safe-to-escape*"